src/main/java/org/example/service/UserService.java

```diff
@@ -0,0 +1,47 @@
+package org.example.service;
+
+import org.example.model.User;
+import org.example.repository.dao.UserRepository;
+import java.util.List;
+
+public class UserService {
+    private final UserRepository userRepository;
+
+    public UserService(UserRepository userRepository) {
+
+        this.userRepository = userRepository;
+    }
+
+
+    public User findById(int id) {
+
+        return this.userRepository.findById(id);
```

Comment:
what is there is no user with the id?

Reply:
> what is there is no user with the id?

As I see I don't use this method in my code, it rather a additional instrumet to implement it in future, so I didn't write handling for this situation. I have the same method for Game and for it I wrote method which doesn't allow you to choose inccorrect id. Can I use 'Optional' here, to avoid mismatch here?